eng/pipelines/docindex.yml

```diff
@@ -49,13 +50,23 @@ jobs:
       - template: /eng/common/pipelines/templates/steps/set-daily-docs-branch-name.yml
         parameters:
           DailyBranchVariableName: DailyDocsBranchName
+      # Sync docs fork repo onboarding files/folders
+      - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
```

Comment:
@sima-zhu @danieljurek just making sure you saw my other question. Instead of doing another clone and having another repo could we just add a remote to the azure-sdk for instead? I believe things should work with that approach without requiring a duplicated repo and need for a new variable pointing to that location.

Reply:
I think it is doable. Sorry for missing the comments above. I will change as you suggested.

Reply:
After second thought, I think two clones here is not a big cost. 
We use the sparse-checkout on `/metadata/`, `package.json`. These files are small chunk of files in docs repo.

However, use remote and new branch to track `azure-sdk\azure-docs-sdk-java` might download the diff of `docs-ref-autogen`, together with `matadata`/ and `package.json`. That's much heavy than clone

Reply:
Talked offline. 
Will proceed with remote to azure-sdk as it also tracking on diff on sparse clone. Still better than two clones.